source/core/GenericFunctions.cpp

```diff
@@ -31,25 +31,38 @@ ConstCStr CopyProcName(void *pSource, void *pDest, TypeRef elemType)
     Int32 alignment = elemType->TheTypeManager()->AQAlignment(aqSize);
     if (((uintptr_t)pSource % alignment != 0) || ((uintptr_t)pDest % alignment != 0))
         return nullptr;
-    switch (aqSize) {  // copy sizes go to 32 bytes just for efficiency
-        case 1:     return "Copy1";   break;
-        case 2:     return "Copy2";   break;
-        case 4:     return "Copy4";   break;
-        case 8:     return "Copy8";   break;
-        case 16:    return "Copy16";  break;
-        case 32:    return "Copy32";  break;
-        default:    return nullptr;      break;
+
+    switch (aqSize) {  // copy sizes go to 32 bytes for efficiency
```

Comment:
I think the original was compact but still readable.  OK with removing the breaks.